ports/qemu-riscv/test_main.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2014 Ilya Dmitrichenko
```

Comment:
This appears to be a copy of `qemu-arm/test_main.c` - which is fine. But perhaps a comment (in both files?) to mention the other in case a change should affect both?

Or maybe finding somewhere common to store it would be worthwhile?

Reply:
It's not a copy, the test heap size is doubled.

Reply:
Eventually we would like to make some common code to share among the qemu ports, but for now this copy is OK.